### ISSUE TITLE
Incompatibility with PyCharm / VSC Pylance

### ISSUE THREAD

**Issue by MiloParigi** [henribru/google-ads-stubs#16]
Following the first issue of this repository #1, I've discovered that my import failures on Pycharm and VSC where coming from the editor themselves and not this package.

For reference, here are the opened tickets on the subject for PyCharm:
- https://youtrack.jetbrains.com/issue/PY-46104
- https://youtrack.jetbrains.com/issue/PY-55589/Support-pytyped-partial-stubs-pep-561
- https://youtrack.jetbrains.com/issue/PY-56217/Partial-stubs-are-not-handled-properly

I was unable to locate similar issues for Pylance. If someone have it working in Visual, please let me know :)

As it seems unlikely that it will be fixed in the near future, would you accept a PR with a note on the IDEs current limitation and non implementation of PEP-561 in the 'Caveats' section of the package README ? It seems to me like a major pain point for users of this package, and something I would like to know before using it.

Thanks !

**Comment by henribru** [comment]
There have been some changes to how this is handled since #1, so assuming you're on a recent version the reason it doesn't work now is slightly different. At least in VS Code's case, this is due to a bug in `types-protobuf` (which is a dependency of this package):
https://github.com/microsoft/pyright/issues/2113
https://github.com/python/typeshed/issues/5800

Unfortunately the [fix](https://github.com/python/typeshed/pull/6106) for this was reverted. As a workaround for nowyou can either delete `types-protobuf` or remove the `__init__.pyi` file it erroneously adds: https://github.com/microsoft/pyright/issues/2113#issuecomment-1196989955. I've tested this and it seems to fix the issue both in VS Code and Pycharm. If you'd like to add those workarounds to the README I'll be happy to accept it.
